docs/admin-manual/user-priorities-negotiation.rst

```diff
@@ -454,7 +454,10 @@ machine it gets, matchmaking gives it the first idle resource to meet
 its requirements.
 
 This matchmaking cycle continues until the user has received all of the
-machines in their pie slice. The matchmaker then contacts the next
+machines in their pie slice. If there is a per-user ceiling defined
+with the *condor_userprio* -setceil command, and this ceiling is smaller
+than the pie slice, the user gets only up to their ceiling number of
+cores.  The matchmaker then contacts the next
```

Comment:
Saying "number of cores" here is inconsistent with the immediately preceding wording referring to "machines", although that in itself is inconsistent and switches from "resources" to "machines" in the middle of the previous paragraph.  Do we not specify that the negotiator actually operates in units of `SlotWeight` anywhere?